nautobot_ssot/migrations/0007_replace_dashed_custom_fields.py

```diff
@@ -0,0 +1,53 @@
+from django.db import migrations
+from nautobot.extras.utils import FeatureQuery
+
+CF_KEY_CHANGE_MAP = {
+    "servicenow_last_synchronized": "ssot-synced-to-servicenow",
+    "last_synced_from_sor": "ssot-synced-from-ipfabric",
+    "ipfabric_site_id": "ipfabric-site-id",
+    "ssot_synced_to_infoblox": "ssot-synced-to-infoblox",
```

Comment:
Sorry to keep nitpicking, but would it make sense to change `servicenow_last_synchronized` to `ssot_synced_to_servicenow` to be consistent with `ssot_synced_to_infoblox`?

Reply:
Done.